### ISSUE TITLE
PillContainer/Combobox: All pills should be aria-selected

### ISSUE THREAD

**Issue by interactivellama** [salesforce/design-system-react#1659]
When pills are present, then our widget is in multiple select mode, thus aria-selected does not follow focus, but informs the user of data section (which is all the pills in the "selected listbox").

Per this spec
https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/listbox/listbox.html#rps_label
